docker/Dockerfile

```diff
@@ -90,7 +90,6 @@ WORKDIR ${HOME}
 COPY --from=base ${HOME} .
 
 # Setup Conda environment
-ENV PATH="${HOME}/conda/bin:${PATH}"
```

Comment:
This change happend `python: not found` error when I built full environment.

Reply:
Oops. We need this line after since I'm pulling from Nvidia base